neptune/new/handler.py

```diff
@@ -144,5 +258,166 @@ def pop(self, path: str, wait: bool = False) -> None:
+
+        Available for following field types (`Field types docs page`_):
+            * `FileSet`
+
+        Args:
+            paths (str or collection of str): `Path` or paths to files or folders to be deleted.
+                Note that these are paths relative to the FileSet itself e.g. if the `FileSet` contains
+                file `example.txt`, `varia/notes.txt`, `varia/data.csv` to delete whole subfolder you would pass
+                varia as the argument.
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server.
+                This makes the call synchronous.
+                Defaults to `None`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def download(self, destination: str = None, wait: bool = True) -> None:
+        """Downloads the stored file or files to the working directory or specified destination.
+
+        Available for following field types (`Field types docs page`_):
+            * `File`
+            * `FileSeries`
+            * `FileSet`
+
+        Args:
+            destination (str, optional): Path to where the file(s) should be downloaded.
+                If `None` file will be downloaded to the working directory.
+                If `destination` is a directory, the file will be downloaded to the specified directory with a filename
+                composed from field name and extension (if present).
+                If `destination` is a path to a file, the file will be downloaded under the specified name.
+                Defaults to `None`.
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server.
+                This makes the call synchronous.
+                Defaults to `None`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
+        """
+        raise NeptuneException('Should be never called.')
+
+    def download_last(self, destination: str = None, wait: bool = True) -> None:
+        """Downloads the stored file or files to the working directory or specified destination.
+
+        Available for following field types (`Field types docs page`_):
+            * `FileSeries`
+
+        Args:
+            destination (str, optional): Path to where the file(s) should be downloaded.
+                If `None` file will be downloaded to the working directory.
+                If `destination` is a directory, the file will be downloaded to the specified directory with a filename
+                composed from field name and extension (if present).
+                If `destination` is a path to a file, the file will be downloaded under the specified name.
+                Defaults to `None`.
+
+        .. _Field types docs page:
+           https://docs.neptune.ai/api-reference/field-types
```

Comment:
Let's link explicitly to https://docs.neptune.ai/api-reference/field-types#download_last